lib/connect4.py

```diff
@@ -13,23 +15,40 @@ def start(self):
                 player = 2
             else:
                 player = 1
-            player_choice = int(input("Make your move"))
-            self.move(player, player_choice - 1)
-            self.show_board()
+            self.move(player)
             not_over = not self.gameover()
             turn += 1
+
         if not_over == True:
             print("It's a Draw!")
         else:
+            self.show_board()
             print(f'Player {player} wins')
 
     def show_board(self):
-        return self.game_board.show()
+        string = ""
+        i = len(self.game_board.game_board)-1
+        while i > -1:
+            row = self.game_board.game_board[i]
+            for token in row:
+                string +=  f'|{token}|'
+            string += "\n"
+            i -= 1
+        print(string)
 
     def gameover(self):
         return self.game_board.moves.check_win()
 
 
-    def move(self, player, legal_move_index):
-        self.game_board.make_move(player, legal_move_index)
-# comment for commit
+    def move(self, player):
+        self.show_board()
```

Comment:
Need to refactor out the nested if statements in connect4.move()

Reply:
Will refactor after feature freeze.